crates/symbolicator-service/src/types/mod.rs

```diff
@@ -698,6 +698,17 @@ pub struct JsFrame {
 pub struct JsFrameData {
     #[serde(skip_serializing_if = "Option::is_none")]
     pub sourcemap: Option<String>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub resolved_with: Option<ResolvedWith>,
+}
+
+#[derive(Debug, Clone, Copy, Deserialize, Serialize, PartialEq, Eq)]
+#[serde(rename_all = "kebab-case")]
+pub enum ResolvedWith {
+    DebugId,
+    Release,
+    ReleaseOld,
+    Scraped,
```

Comment:
```suggestion
    Scraping,
```

just a stylistic nit :-)